k8s/ietfweb/local.py

```diff
@@ -0,0 +1,114 @@
+# Copyright The IETF Trust 2007-2024, All Rights Reserved
+# -*- coding: utf-8 -*-
+
+from email.utils import parseaddr
+import os
+
+def _multiline_to_list(s):
+    """Helper to split at newlines and conver to list"""
+    return [item.strip() for item in s.split("\n")]
+
+
+DEFAULT_FROM_EMAIL = "donotreply@ietf.org"
+SERVER_EMAIL = "donotreply@ietf.org"
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = os.environ.get("IETFWWW_EMAIL_HOST", "localhost")
+EMAIL_PORT = int(os.environ.get("IETFWWW_EMAIL_PORT", "2025"))
+
+# Default to "development". Production _must_ set IETFWWW_SERVER_MODE="production" in the env!
+SERVER_MODE = os.environ.get("IETFWWW_SERVER_MODE", "development")
```

Comment:
Curious - what does `SERVER_MODE` actually do?

Reply:
I think this is a DT thing. I'll remove this.

Reply:
Addressed with https://github.com/ietf-tools/www/pull/453/commits/aead1e09ab0966e0e818728600d1ab349a1cbb94